piet/src/text.rs

```diff
@@ -68,6 +100,9 @@ pub struct HitTestPoint {
 }
 
 /// return values for [`hit_test_text_position`](../piet/trait.TextLayout.html#tymethod.hit_test_text_position).
+/// - `metrics.text_position` will give you the text position.
+/// - `point` is the width offset of the leading edge of the grapheme cluster containing the text
```

Comment:
I would say something like, "the point's `y` value is the position of the leading edge of ..."

Reply:
thanks, that's better wording (although I think it's the `x` in this case)